CONTRIBUTING-gitflow.md

```diff
@@ -1,95 +1,149 @@
-# Working by Forking
-Just head over to the GitHub page and click the "Fork" button. It's just that simple. Once you've done that, you can use your favorite git client to clone your repo or just head straight to the command line:
-
-## Clone your fork to your local machine
-```
-git clone git@github.com:USERNAME/FORKED-PROJECT.git
-```
-Keeping Your Fork Up to Date
-While this isn't a necessary step, if you plan on doing anything more than a tiny quick fix, you'll want to make sure you keep your fork up to date by tracking the original "upstream" repo that you forked earlier. To do this, you'll need to add a remote:
-
-## Add 'upstream' repo to list of remotes
-```
-git remote add upstream https://github.com/meshery/meshery.git
-``` 
+<p align="center">
+<img src="https://img.shields.io/badge/open%20source-%E2%9D%A4%EF%B8%8F-violetgreen">
+<img src="https://img.shields.io/badge/PRs-welcome-green">
+</p>
+
+## Setting Project on Local System :-
+
+<ul>
+<li><h3> Fork the Repository of Project</h3></li><br>
+
+<p><img src="fork.png" align="center"></p>
+<br><br>
+( You will see this on Top Right of Github Repository !)<br><br>
+<li><h3>Clone your fork to your local machine</h3></li><br>
+<img width="50%" align="center"  src="clone.png"><br><br>
+
+( Click on the Green Code button and Copy the link `https://github.com/........` )</h3></li><br><br>
+
+<li><h3>Open Git bash where you want to clone the project ( Avoid On Desktop )</h3></li>
+
+<li><h3>Run Command</h3></li>
+
+`git clone link`
+
+(In Place of link paste the link you copied)
+
+<h3>Project Cloned in System</h3>
+<br><br>
+
+<li><h3>Add 'upstream' repo to list of remotes</h3></li><br>
+Keeping Your Fork Updated
+While it's not required, if you plan on making anything more than a fast patch, you'll want to keep your fork up to date by tracking the original "upstream" source you forked previously. You'll need to add a remote to accomplish this:<br><br>
+
+`git remote add upstream https:/github.com/meshery/meshery.git`
+<br><br>
 ("meshery" is used as the example repo. Be sure to reference the _actual_ repo you're contributing to e.g. "meshery-linkerd").
+<br>
+
+<li><h3>Verify the new remote named 'upstream'</h3></li>
+
+`git remote -v`
+<br>
+
+<li><h3>Fetch from upstream remote</h3</li>
+<br>
+You'll need to fetch the upstream repo's branches and newest commits to bring them into your repository whenever you wish to update your fork with the latest upstream changes:
+<br>
+
+`git fetch upstream`
+<br><br>
+
+<li><h3>View all branches, including those from upstream</h3></li>
+<br>
+
+`git branch -va`
+<br>
```

Comment:
Not sure we need this level of detail for a trivial thing (`git branch -va` is not fundamental regards to gitflow itself) that git [command] --help can tell contributors (or even [git reference docs](https://git-scm.com/docs)). can this be removed please @Nandani-Paliwal? In spirit of keeping this readme as short as short & concise as possible.

Reply:
I'd be open to adding a link to the [git reference docs](https://git-scm.com/docs) in a separate section at the bottom called "References" so contributors reference it as needed.